app/controllers/v0/disability_compensation_forms_controller.rb

```diff
@@ -51,8 +51,18 @@ def submit_all_claim
       saved_claim = SavedClaim::DisabilityCompensation::Form526AllClaim.from_hash(form_content)
       saved_claim.save ? log_success(saved_claim) : log_failure(saved_claim)
       submission = create_submission(saved_claim)
-
-      jid = submission.start
+      # if jid = 0 then the submission was prevented from going any further in the process
+      jid = 0
```

Comment:
Is this a convention you're establishing, or the current rule? Was curious if multiple jobs in the db with the same `jid` would cause issues, but I can probably confirm this